.github/workflows/devcontainer-build.yaml

```diff
@@ -6,7 +6,7 @@ on:
       - main
 
     paths:
-    - '.devcontainer/Dockerfile'
+    - 'helm-validate/Dockerfile'
```

Comment:
I noticed that we have duplicates between .devcontainer/.devcontainer.json and helm-validate/.devcontainer/devcontainer.json do we need both?

Reply:
The .devcontainer.json at the root is for the person to point vs-code to open which docker/image locally. the devcontainer.jsons in both helm-validate and dind-k3d is for the github actions workflow to point towards.